dvc/exceptions.py

```diff
@@ -326,3 +328,16 @@ def __init__(self, path_info):
                 format_link("https://man.dvc.org/config#cache"),
             )
         )
+
+
+class AddOverlappingPathsError(DvcException):
+    def __init__(self, parent, overlapping_out):
+        super().__init__(
+            "Cannot add '{out}', because it is overlapping with other DVC "
+            "tracked output: '{parent}'.\nTo include '{out}' in '{parent}', "
+            "run 'dvc commit {parent_stage}'".format(
+                out=overlapping_out,
+                parent=parent,
+                parent_stage=parent.stage.relpath,
+            )
+        )
```

Comment:
Let's do it a bit differently. Let's only keep `OverlappingOutputPathsError`, but raise it with specific message instead of hardcoding it into the exception class itself. That way we won't spawn new entities and it will be nicely catchable from outside if needed :slightly_smiling_face:

Reply:
Yeah, makes sense